pkg/sensors/tracing/generickprobe.go

```diff
@@ -658,6 +658,15 @@ func createGenericKprobeSensor(
 		Policy:    polInfo.name,
 		Namespace: polInfo.namespace,
 		DestroyHook: func() error {
+			for _, id := range ids {
+				if gk, err := genericKprobeTableGet(id); err == nil {
+					selectors.ClearKernelSelectorState(gk.loadArgs.selectors.entry)
+					if gk.loadArgs.retprobe {
+						selectors.ClearKernelSelectorState(gk.loadArgs.selectors.retrn)
+					}
+				}
+			}
+
```

Comment:
We do `genericKprobeTable.RemoveEntry(id)` in the code below.
Why do we also need to clear the selectors? Do we hold other referrences of these entries somewhere else?
Because I would expect GC to clean the selector state once there are no more references.

Reply:
Now, this is ready for review.